storage/rocksdb/CMakeLists.txt

```diff
@@ -35,6 +35,12 @@ IF ((CMAKE_CXX_COMPILER_ID STREQUAL GNU) AND
   RETURN ()
 ENDIF ()
 
+# Do not treat warnings as errors for gcc-8+ or clang-8+
```

Comment:
This still results in noisy build log - why not disable selected warnings instead?